packages/ui/src/components/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { Typography, BoxProps } from './Typography';
```

Comment:
Add a new line after imports. It should be fixed automatically by ESLint, so you might have some issue with its configuration across your IDE

Reply:
Yes, I had a problem with the IDE and ESLint but I think I solved it;-)